client/components/date-range/README.md

```diff
@@ -0,0 +1,50 @@
+DateRange
+==========
+
+React component used to display a Date Range.
```

Comment:
Nit - lets add `display and select` here as selecting the range is what really makes this component :)

Reply:
Fair. Had already updated but have re-updated again to reinstate the `display` bit as well.